src/Microsoft.DotNet.Build.Tasks/Targets/Build.Common.targets

```diff
@@ -84,6 +84,11 @@
   -->
   <Import Project="$(MSBuildThisFileDirectory)versioning.targets" Condition="'$(ExcludeVersioningImport)'!='true'" />
 
+  <!--
+    Import the localization target
+  -->
+  <Import Project="$(MSBuildThisFileDirectory)localization.targets" Condition="'$(ExcludeLocalizationImport)'!='true' And Exists('$(MSBuildProjectDirectory)\MultilingualResources\$(MSBuildProjectName).de.xlf')" />
```

Comment:
Ah, that makes sense.  Giving projects the way to override the Exists check makes sense.

I think looking for de.xlf is fine.  Really, all 9 .xlf files and all 9 localized .resx files are necessary; otherwise the build will failed.  But as we discussed earlier in the other thread here, I think it's fine to just check for one of them.